lib/versioning/loose/generic.ts

```diff
@@ -81,10 +81,11 @@ export const comparer = (
     return versions.find(v => equals(v, range)) || null;
   }
   function getNewValue(
-    _currentValue: string,
-    _rangeStrategy: RangeStrategy,
-    _fromVersion: string,
-    toVersion: string
+    { toVersion }: NewValueConfig = {
+      currentValue: null,
+      rangeStrategy: null,
+      toVersion: arguments[0],
```

Comment:
@souravdasslg can this be refactored too? I find this syntax really confusing

Reply:
Sure. Ill do that.